composer.json

```
@@ -27,7 +27,7 @@
         "doctrine/collections": "^1.5 || ^2.0",
         "doctrine/event-manager": "^1.0 || ^2.0",
         "doctrine/instantiator": "^1.1 || ^2",
-        "doctrine/persistence": "^3.2 || ^4",
+        "doctrine/persistence": "^3.2 || ^4@dev",
```

After the release of `doctrine/persistence 4.1.0`
```suggestion
        "doctrine/persistence": "^3.2 || ^4",
```